### PR TITLE
Add Contact Form to Portfolio Website

### PR DESCRIPTION
**Pull request by Andyveloper**
## Changes & Additions
- Add contact form as per requirements
- Form types used properly
- Comply with exercise requirements
- Use Formspree to receive form data
- Linters passed properly
- Add comments for clarity
- Follow Figma template to the detail.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -178,5 +178,35 @@ <h2 class="lang-title">Skills</h2>
...
+      <button type="submit" class="main-button">Get in touch</button>
+    </form>
+    <div id="marginTop">
+      <ul class="iconsBot">
+        <li><img src="./img/social/dark-icons/Github.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Linkedin.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Angelist.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Twitter.png" alt=""></li>
+        <li><img src="./img/social/dark-icons/Medium.png" alt=""></li>
+      </ul>
```
Comment (youmari) [inline index.html]: - [ ] [OPTIONAL] these are links to your social media so you should add an `<a>`  tag to all and your social media URL

File: index.html [hunk 2]
```diff
@@ -178,5 +178,35 @@ <h2 class="lang-title">Skills</h2>
...
+          <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+        </li>
+        <li class="formInput">
+          <input type="email" id="mail" name="email" placeholder="E-mail address" required>
+        </li>
+        <li class="formInput" id="no-border">
+          <textarea  class="textarea1" name="message" maxlength="500" required placeholder="Enter text here"></textarea>
+        </li>
+      </ul>
+      <button type="submit" class="main-button">Get in touch</button>
```
Comment (youmari) [inline index.html:198]: please try to stick to design by implementing the button effect on hover and on pressed as in the Figma design 
image

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Zeraltz @Daf98, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Descriptive readme file 
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @Zeraltz , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**